<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLineEdit" name="_PwdEditor">
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2" alignment="Qt::AlignHCenter">
    <widget class="QDialogButtonBox" name="_ButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="_NameEditor">
     <property name="text">
      <string>admin</string>
     </property>
     <property name="placeholderText">
      <string>admin</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="_AddrEditor">
     <property name="text">
      <string>127.0.0.1</string>
     </property>
     <property name="placeholderText">
      <string>127.0.0.1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Address:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QToolButton" name="_RegisterBtn">
     <property name="text">
      <string>Register</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>_ButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>LoginDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
